streams/src/test/java/org/apache/kafka/streams/state/internals/ThreadCacheTest.java

```diff
@@ -285,6 +285,25 @@ public void shouldPeekAndIterateOverRange() {
         assertEquals(5, bytesIndex);
     }
 
+    @Test
+    public void testPrefixScan() {
```

Comment:
If we remove `prefixScan()`, you can remove this test, but we need a test for `range()` that excludes the end limit.